common/components/Header/components/Navigation.tsx

```diff
@@ -5,6 +5,7 @@ import './Navigation.scss';
 
 interface Props {
   color?: string | false;
+  unsupportedTabs?: string[];
```

Comment:
Would prefer to create a union of all tabs and so we can get some type safety around what tabs will be unsupported:
```
type TABS = 'CREATE' | 'SWAP' | 'CONTRACT' ...

const unsupportedTabs: TABS[] = ['SWAP'];
```

Reply:
Would prefer enums here (can be string enums if needed) such as 
```ts
enum TAB {
 CREATE,
 SWAP, 
}
const unsupportedTabs: TAB[] = [TAB.CREATE]
```